pkg/network/node/vnids.go

```diff
@@ -136,6 +136,21 @@ func (vmap *nodeVNIDMap) getVNID(name string) (uint32, error) {
 	return 0, fmt.Errorf("failed to find netid for namespace: %s in vnid map", name)
 }
 
+func (vmap *nodeVNIDMap) findDuplicateNetID(namespace string, netID uint32) (bool) {
+	vmap.lock.Lock()
+	defer vmap.lock.Unlock()
+
+	for name, id := range vmap.ids {
```

Comment:
`vmap` has indexes both by name and by id; you can call `vmap.GetNamespaces(vnid)` to get the list of namespaces with a given vnid (which will hopefully be either empty, or just a one-element list containing the namespace you were expecting)